test/test-find-python.js

```diff
@@ -10,19 +10,15 @@ const PythonFinder = findPython.test.PythonFinder
 require('npmlog').level = 'warn'
 
 test('find python', function (t) {
-  t.plan(4)
+  t.plan(5)
 
   findPython.test.findPython(null, function (err, found) {
     t.strictEqual(err, null)
     var proc = execFile(found, ['-V'], function (err, stdout, stderr) {
       t.strictEqual(err, null)
-      if (/Python 2/.test(stderr)) {
-        t.strictEqual(stdout, '')
-        t.ok(/Python 2/.test(stderr))
-      } else {
-        t.ok(/Python 3/.test(stdout))
-        t.strictEqual(stderr, '')
-      }
+      t.notOk(/Python 2/.test(stderr))
+      t.ok(/Python 3/.test(stdout))
+      t.strictEqual(stderr, '')
```

Comment:
I just noticed I kept redundant checks in this test:

`t.notOk(/Python 2/.test(stderr))` is redundant to
`t.strictEqual(stderr, '')` two lines later.

With permission from the maintainers/reviewers, I'd want to delete one of them, preferably the first one.

<details>

For context:
- `python2 -V` outputs the Python version on `stderr`, like this: `'Python 2.7.16'`.
- `python3 -V` outputs the version on `stdout` like this: `'Python 3.8.2'`.

This test is just checking version strings when printing the version of Python that `findPython()` found.

Example script, showing that Python 2 and Python 3 print to `stderr` and `stdout` respectively:

```js
const execFile = require('child_process').execFile

execFile('python2', ['-V'], function (err, stdout, stderr) {
  console.log('--- python2 -V output ---')
  console.log('stdout is: ' + stdout)
  console.log('stderr is: ' + stderr)
})

execFile('python3', ['-V'], function (err, stdout, stderr) {
  console.log('--- python3 -V output ---')
  console.log('stdout is: ' + stdout)
  console.log('stderr is: ' + stderr)
})
```

Output:

```
--- python2 -V output ---
stdout is: 
stderr is: Python 2.7.16

--- python3 -V output ---
stdout is: Python 3.8.2

stderr is: 
```

</details>

Reply:
You do not need to wait for permission.  If the test is redundant that it can be eliminated.